docs/changelog.rst

```diff
@@ -1,12 +1,14 @@
 Changelog
 =========
 
+12.0 (?)
```

Comment:
should this be updated with a date or not yet?

Reply:
Once released.